microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -79,126 +79,148 @@ In **Guest** mode, autofill is not available and new autofill entries aren't add
 
-You can access Collections from an InPrivate window while signed in with a work or school account.
 
+<!-- ====================================================================== -->
+## Copilot in Microsoft Edge sidebar
 
-<!-- ------------------------------ -->
-#### Signed-out
+In Copilot in the Microsoft Edge sidebar, you can ask complex questions, find comprehensive answers, get summarized information, and find inspiration to build upon, in a side-by-side view, with no need to flip between tabs.
 
-When you don't sign in to Microsoft Edge, all collections data is stored locally on the device, organized per Microsoft Edge profile.
+To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar.
 
-You can access Collections from an InPrivate window when using a signed-out profile.
 
+<!-- --------------------------------------------------------------------- -->
+#### Data and consent used by Copilot in Microsoft Edge 
 
-<!-- ====================================================================== -->
-## Compose
+Microsoft Edge determines what data to send to Copilot based on your **query** and your consent to share data with Microsoft.  For questions that don't need **browsing context**, such as "Help me plan a trip to Cannon Beach", Microsoft Edge shares the URL, the page title, your query, and previous conversation history to help Copilot answer your query effectively.
 
-In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
+For questions that need **browsing context**, such as when you ask Copilot to summarize a large page of text, Microsoft Edge asks for your permission to access page information:
 
-![Compose UI](./index-images/compose.png)
+![Copilot asking for consent to access the page content in Microsoft Edge](./index-images/consent-for-contextual-browsing.png) <!-- Update Bing chat screenshot-->
+
+When you grant permission to share page information, Microsoft Edge sends the full **browsing context** in addition to your **query** and previous **conversation history** to Copilot, to help generate a meaningful **response**.
 
-To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 
+To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot**, and then turn on or off the **Allow access to any webpage or PDF** toggle.
```

Comment:
```suggestion
To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot**, and then toggle the **Allow access to any webpage or PDF** setting.
```

Reply:
Apple style guide: 
https://support.apple.com/guide/applestyleguide/t-apsg841c3645/1.0/web/1.0#apd37a491bb67054 - "_toggle_ (as verb): Don’t use in user materials; instead, say turn on or off, switch between, or whatever wording is appropriate in the context."